docker/docker-compose-mock-envoy.yml

```diff
@@ -0,0 +1,7 @@
+version: '3.8'
+services:
+  mock-envoy:
+    container_name: mock-envoy
+    image: europe-west2-docker.pkg.dev/ons-ci-int/int-docker-snapshot/mock-envoy:update-dependencies
```

Comment:
Consider using tags so that when we do `docker sp` we get a human readable image name